### PR TITLE
[CROSSDATA-349] Close disconnected session

### PR DESCRIPTION
**Pull request by pfcoperez**
## Description

This PR leverages the KeepAlive building block to keep track of client opened sessions (at Crossdata Driver). 

* Centralices session management.
* Closes sessions as jobs are cancelled after an inactive session has expired.
* Terminates jobs belonging to explicitly closed sessions.
* Removes few vestigial code snippets.

### Testing
- [x] Unit, integration tests

### CODE REVIEW

File: driver/src/main/scala/com/stratio/crossdata/driver/actor/SessionBeaconActor.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  def props(
+             sessionId: UUID,
+             period: FiniteDuration,
+             clusterClientActor: ActorRef,
+             clusterPath: String): Props =
+    Props(new SessionBeaconActor(sessionId, period, clusterClientActor, clusterPath))
+
+}
+
+class SessionBeaconActor private (
```
Comment (miguel0afd) [inline driver/src/main/scala/com/stratio/crossdata/driver/actor/SessionBeaconActor.scala]: I would comment the functionality of this class

**Comment by miguel0afd** [comment]
Nice! 👍

File: server/src/main/scala/com/stratio/crossdata/server/actors/ServerActor.scala [hunk 2]
```diff
@@ -93,9 +89,9 @@ class ServerActor(cluster: Cluster, sessionProvider: XDSessionProvider, serverAc
   }
 
   private def checkSubscriptions(pendingTopics: Set[String]): Unit =
-    if (pendingTopics.isEmpty)
+    if (pendingTopics.isEmpty) {
       context.become(ready(State(Map.empty)))
-    else
+    } else
```
Comment (darroyocazorla) [inline server/src/main/scala/com/stratio/crossdata/server/actors/ServerActor.scala]: missing curly brackets?

**Comment by darroyocazorla** [comment]
:+1:

**Pull request merged**